osm2streets/src/lanes/classic.rs

```diff
@@ -594,7 +594,8 @@ fn osm_separation_type(x: &String) -> Option<BufferType> {
     match x.as_ref() {
```

Comment:
Just noting, we should try and sync with https://wiki.openstreetmap.org/wiki/Proposed_features/separation#Typical_separation_values and taginfo at some point. I wrote `BufferType` quite a while ago at this point!